src/util/memory_units.cpp

```diff
@@ -80,7 +80,7 @@ std::string memory_sizet::to_string() const
     {
       out << ' ';
     }
-    out << kib << si_kibitye_symbol;
+    out << kib << si_kibibyte_symbol;
```

Comment:
memory_unit changes should be in previous commit

Reply:
Refactored and sqashed.